tests/unit_and_functional/conftest.py

```diff
@@ -55,15 +56,16 @@ def setup_bodywork_test_project(
     try:
         run(['git', 'init'], cwd=project_repo_location, check=True)
         run(['git', 'add', '-A'], cwd=project_repo_location, check=True)
-        run(['git', 'commit', '-m', '"test"'], cwd=project_repo_location, check=True)
-        run(['mkdir', bodywork_output_dir], check=True)
+        run(['git', 'commit', '-m', '"test"'], cwd=project_repo_location, check=True, capture_output=True)
```

Comment:
Could you please add `encoding='utf-8` to the options please?

This will stop the output being written as bytes. You're basically correcting my original work here - sorry...

Reply:
No worries, done!